likeminds-chat-reactnative-integration/ChatSX/screens/FIleUpload/index.tsx

```diff
@@ -59,13 +59,15 @@ interface FileUploadProps {
   handleCamera?: () => void;
   handleDoc?: () => void;
   onEdit?: () => void;
+  conversationMetaData?: any;
```

Comment:
define types.

Reply:
Made this to `Record<string, any>` as this is meta data object that we will pass